treasurenetprotocol/treasurenet
# Add root_test file to test the ability to generate keys from binary

Dear team members,


I hereby submit a Pull Request to merge the newly written test file root_test into main. This file was created to test the functionality of generating keys from binary, and it will help us ensure the correctness and stability of this feature.


Change content


Added root_test file under the main branch.

The root_test file contains test cases for the "generate key from binary" feature.


testing procedure


Ensure that all necessary dependencies have been installed in your local environment.

Pull the latest main branch to the local location.

Run the test cases in the root_test file to ensure that all tests pass.
